cabal.project

```diff
@@ -34,9 +30,16 @@ source-repository-package
   --sha256: 0pb7bg0936fldaa5r08nqbxvi2g8pcy4w3c7kdcg7pdgmimr30ss
 
 constraints:
-  ip < 1.5,
   hedgehog >= 1.0,
-  bimap >= 0.4.0
+  bimap >= 0.4.0,
+  bytestring >= 0.11
```

Comment:
The bytestring constraint should go in the cabal file of the package that needs it, perhaps? The others could do with comments so we know why we need them and when we can remove them?